en/computer_vision/obstacle_avoidance.md

```diff
@@ -18,146 +28,59 @@ The desired route comes from a [ROS](http://dev.px4.io/en/ros/) node running on
-
-The tested companion computer platform is [Intel Aero](../complete_vehicles/intel_aero.md) running either the *local_planner* or *global_planner* avoidance software. Object avoidance can also be tested using Gazebo.
-The set up for both is as described in the [Intel Aero > Obstacle Avoidance](../complete_vehicles/intel_aero.md#obstacle-avoidance) and in the [PX4/avoidance](https://github.com/PX4/avoidance#obstacle-detection-and-avoidance) Github repo.
+## Mission Mode Avoidance {#mission_mode}
 
+PX4 supports obstacle avoidance in [Mission mode](../flight_modes/mission.md), using avoidance software running on a separate companion computer.
 
 ### Mission Progression
 
 Mission behaviour with obstacle avoidance enabled is *slightly different* to the original plan.
 
 The difference when avoidance is active are:
 - A waypoint is "reached" when the vehicle is within the acceptance radius, regardless of its heading.
-  - This differs from normal missions, in which the vehicle must reach a waypoint with a certain heading (i.e. in a "close to" straight line from the previous waypoint). This constraint cannot be fulfilled when bstacle avoidance is active because the obstacle avoidance algorithm has full control of the vehicle heading, and the vehicle always moves in the current field of view. 
+  - This differs from normal missions, in which the vehicle must reach a waypoint with a certain heading (i.e. in a "close to" straight line from the previous waypoint). This constraint cannot be fulfilled when obstacle avoidance is active because the obstacle avoidance algorithm has full control of the vehicle heading, and the vehicle always moves in the current field of view. 
 - PX4 starts emitting a new current/next waypoint once the previous waypoint is reached (i.e. as soon as vehicle enters its acceptance radius).
-- If a waypoint is *inside* an obstacle it may unreachable (and the mission will be stuck). 
+- If a waypoint is *inside* an obstacle it may unreachable (and the mission will be stuck).
   - If the vehicle projection on the line previous-current waypoint passes the current waypoint, the acceptance radius is enlarged such that the current waypoint is set as reached
   - If the vehicle within the x-y acceptance radius, the altitude acceptance is modified such that the mission progresses (even if it is not in the altitude acceptance radius).
 - The original mission speed (as set in *QGroundControl*/PX4) is ignored.
   The speed will be determined by the avoidance software:
   - *local planner* mission speed is around 3 m/s.
   - *global planner* mission speed is around 1-1.5 m/s.
 
-If PX4 stops receiving setpoint updates then obstacle avoidance will be disabled, and the mission continues under normal PX4 [Mission mode](../flight_modes/mission.md) control. 
+If PX4 stops receiving setpoint updates then obstacle avoidance will be disabled, and the mission continues under normal PX4 [Mission mode](../flight_modes/mission.md) control.
```

Comment:
Now we have proper failsafe. The vehicle will go into Loiter if the setpoints are older than half a second

Reply:
I see that it's covered below. Maybe we can remove this sentence

Reply:
Thanks. I corrected but kept the sentence. It is a level of duplication, but I think users will appreciate knowing about this at this level rather than hidden in the other doc.